ElectronClient/app.js

```diff
@@ -1188,7 +1188,7 @@ class Application extends BaseApplication {
 		const selectedNoteIds = state.selectedNoteIds;
 		const note = selectedNoteIds.length === 1 ? await Note.load(selectedNoteIds[0]) : null;
 
-		for (const itemId of ['copy', 'paste', 'cut', 'selectAll', 'bold', 'italic', 'link', 'code', 'insertDateTime', 'commandStartExternalEditing', 'showLocalSearch']) {
+		for (const itemId of ['paste', 'cut', 'bold', 'italic', 'link', 'code', 'insertDateTime']) {
```

Comment:
Why did you remove more than the asked command?

Reply:
I was checking which of the functions worked in viewer mode, as I did not know why they are being disabled. Missed reverting that. Sorry